model/variableSheet.cc

```diff
@@ -30,39 +31,44 @@ using ecolab::cairo::CairoSave;
 namespace minsky
 {
 	
-namespace
-{    
-  struct CroppedPango: public Pango
-  {
-    cairo_t* cairo;
-    double w, x=0, y=0;
-    CroppedPango(cairo_t* cairo, double width): Pango(cairo), cairo(cairo), w(width) {}
-    void setxy(double xx, double yy) {x=xx; y=yy;}
-    void show() {
-      CairoSave cs(cairo);
-      cairo_rectangle(cairo,x,y,w,height());
-      cairo_clip(cairo);
-      cairo_move_to(cairo,x,y);
-      Pango::show();
-    }
-  };
-}    
+  namespace
+  {    
+    struct CroppedPango: public Pango
+    {
+      cairo_t* cairo;
+      double w, x=0, y=0;
+      CroppedPango(cairo_t* cairo, double width): Pango(cairo), cairo(cairo), w(width) {}
+      void setxy(double xx, double yy) {x=xx; y=yy;}
+      void show() {
+        CairoSave cs(cairo);
+        cairo_rectangle(cairo,x,y,w,height());
+        cairo_clip(cairo);
+        cairo_move_to(cairo,x,y);
+        Pango::show();
+      }
+    };
+  }
 
```

Comment:
This class is a candidate for being dried out.

Reply:
Done.